operator/v1/types_network.go

```diff
@@ -123,6 +123,35 @@ type NetworkMigration struct {
 	// changing the MTU for the default network will be rejected.
 	// +optional
 	MTU *MTUMigration `json:"mtu,omitempty"`
+
+	// features contains the features migration configuration. Set this to migrate
+	// feature configuration when changing the cluster default network provider.
+	// +optional
+	Features *FeaturesMigration `json:"features,omitempty"`
```

Comment:
is it better to use `featuresMigration` instead of `features` since features is very generic ?

Reply:
This field is under `migration`. The full-path would be in `spec.migration.features`. Would it be clear enough?